02-microarray/ortholog-mapping_microarray_01_ensembl.Rmd

```diff
@@ -178,36 +178,49 @@ The [HGNC Comparison of Orthology Predictions (HCOP)](https://www.genenames.org/
 In general, an orthology prediction where most of the databases concur would be considered the reliable, and we will use this to prioritize mapping in cases where there is more than one possible ortholog for a gene.
 HCOP was originally designed to show orthology predictions between human and mouse, but has been expanded to include data from 18 genomes, including zebrafish, which we will use in this notebook [@hcop-help].
 
-First, we need to download the file from the server holding the HGNC data.
-Go to this [directory page of the HGNC Comparison of Orthology Predictions (HCOP) files](ftp://ftp.ebi.ac.uk/pub/databases/genenames/hcop/).
+We can automatically download the human zebrafish file we need for this example using `download.file()` command. 
+For this notebook, we want to download the file named `human_zebrafish_hcop_fifteen_column.txt.gz`.
 
-This is where the files that reflect the data provided via the [HGNC database](https://www.genenames.org/) are maintained.
-Ortholog species files with the '6 Column' output returns the raw assertions, Ensembl gene IDs and Entrez Gene IDs for human and one other species, while the '15 Column' output includes additional information such as the chromosomal location, accession numbers and the databases that support the assertions.
+First we'll declare a sensible file path for this. 
 
-*Note:* If you are using Safari (or the above FTP server link does not open in a web browser), you may need to go to the [link for the HCOP search tool](https://www.genenames.org/tools/hcop/) and scroll down to "Bulk Downloads" to choose a file to download.
-Here, you can find the same files you would find at the server linked above.
+```{r}
+# Declare what we want this file to be called once we download it
+zebrafish_hgnc_file <- file.path(
+  data_dir,
+  # Can change what the file will be called locally
+  "human_zebrafish_hcop_fifteen_column.txt.gz"
+)
+```
 
-To download a file, click the file name.
-For this notebook, you will want to download the file named `human_zebrafish_hcop_fifteen_column.txt.gz`.
-If you are using a different dataset, you can replace `zebrafish` in `human_zebrafish_hcop_fifteen_column.txt.gz` with the name of the species you have data for, and click on that file to download.
+Using the file path we just declared, we can use the `destfile` argument to download the file to this directory and use this file name. 
 
-<img src="https://github.com/AlexsLemonade/refinebio-examples/raw/07499927a22ed7aa045245e40d692f8e49d7ba52/template/screenshots/download-hgnc-zebrafish-file.png" width=400>
+```{r}
+download.file(
+  paste0(
+    "ftp://ftp.ebi.ac.uk/pub/databases/genenames/hcop/",
+    # Can replace with the file name of the species conversion you are looking for
```

Comment:
Without looking, I don't think people will be able to guess what the correct file name is. Perhaps restore lines 191-192 for context?

Also, comment width:
```suggestion
    # Replace with the file name of the species conversion you are looking for
```

Reply:
I changed that but didn't push the commit yet.  (the first thing, not the comment thing)